Resources/Prototypes/_SpaceCats/Entities/Objects/Misc/identification_cards.yml

```
@@ -21,3 +21,15 @@
+
+- type: entity
+  parent: IDCardStandard
+  id: SecurityEngineerIDCard
+  name: карта полевого инженера
+  components:
+  - type: Sprite
+    layers:
+    - state: default
+    - state: idsecurityenginer
+  - type: PresetIdCard
+    job: SecurityEngineer
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)